plugin/tms/tms/src/main/java/org/geomajas/gwt2/plugin/tms/client/TmsClient.java

```diff
@@ -59,6 +111,63 @@ public static TmsClient getInstance() {
 	// ------------------------------------------------------------------------
 
 	/**
+	 * Create a map with one of the default profiles.
+	 * 
+	 * @param profile
+	 * @return
+	 */
+	public MapConfiguration createTmsMap(Profile profile) {
+		return createTmsMap(profile, 21);
+	}
+
+	/**
+	 * Create a map with one of the default profiles and a specific number of zoom levels (default = 21).
+	 * 
+	 * @param profile
+	 * @param nrOfZoomLevels
+	 * @return
+	 */
+	public MapConfiguration createTmsMap(Profile profile, int nrOfZoomLevels) {
+		switch (profile) {
+			case GLOBAL_GEODETIC:
+				return createTmsMap(profile, "EPSG:4326", CrsType.DEGREES, new Bbox(-180, -90, 360, 180), 256,
+						nrOfZoomLevels);
+			default:
+				return createTmsMap(profile, "EPSG:3857", CrsType.METRIC, new Bbox(-MERCATOR, -MERCATOR, 2 * MERCATOR,
+						2 * MERCATOR), 256, nrOfZoomLevels);
+		}
```

Comment:
and if the profile is local, with a custom EPSG ? Shouldn't it be "not implemented yet"?